python/ray/ml/session.py

```diff
@@ -0,0 +1,98 @@
+from typing import Optional
+
+import logging
+import os
+import shutil
+
+from ray import tune
+from ray import train
+from ray.ml.checkpoint import Checkpoint
+from ray.tune.function_runner import StatusReporter
+
+logger = logging.getLogger(__name__)
+
+_LOCAL_PATH = "local_path"
+_DATA_DICT = "data_dict"
+
+
+def _get_session():
+    # TODO(xwjiang): Currently the ml session only multiplexes between Tune and Train session.
+    #  We should extract common functionalities if possible.
+    from ray.tune.session import _session as tune_session
+    from ray.train.session import _session as train_session
+    if train_session and tune_session:
+        logger.warning("Expected to be either in tune session or train session but not both.")
```

Comment:
I'm wondering if we're actually always in a train+tune session if using train in tune